include/openthread/srp_client.h

```diff
@@ -201,7 +201,7 @@ typedef void (*otSrpClientAutoStartCallback)(const otSockAddr *aServerSockAddr,
  *
  *  - The SRP client is started - `otSrpClientStart()` is called.
  *  - Host name is set - `otSrpClientSetHostName()` is called.
- *  - At least one host IPv6 address is set - `otSrpClientSetHostName()` is called.
+ *  - At least one host IPv6 address is set - `otSrpClientSetHostAddresses()` is called.
```

Comment:
Need to also increment:

https://github.com/openthread/openthread/blob/ec69ad31f9005e447056f8b5d6a4ac75a343b77a/include/openthread/instance.h#L56

Reply:
thx, incremented instance.h now